apps/nextjs-website/src/lib/strapi/homepage.ts

```diff
@@ -0,0 +1,44 @@
+import * as t from 'io-ts/lib';
+import * as qs from 'qs';
+import { fetchFromStrapi } from './fetchFromStrapi';
+
+const LinkHomepageCodec = t.strict({
```

Comment:
```suggestion
const LinkCodec = t.strict({
```

Reply:
This codec for related links will be used also in overview pages, maybe it could be moved in separte file :thinking:

Reply:
I totally agree with you, but at the same time, I think that at the moment we can keep the codec here, because the scope of this PR is to work on homepage.

I know that this codec will be used by other pages as well, like the `overview` as you said, and I think that these changes are going to be made in the PR that needs to use that codec.

Reply:
Two more cents: in this PR I'm not going to think about how the system is going to be; I just focus on what I need now, that's why you see the codec there.